app/Models/User.php

```diff
@@ -41,4 +48,29 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+
+    public function ahthors()
+    {
+        return $this->hasMany(Author::class);
+    }
+
+    public function comments()
+    {
+        return $this->hasMany(Comment::class);
+    }
+
+    public function bookLikes()
+    {
+        return $this->hasMany(BookLike::class);
+    }
+
+    public function followers()
```

Comment:
https://laravel.com/docs/10.x/eloquent-relationships#many-to-many-polymorphic-relations